### DISCUSSION TITLE
Zigbee2MQTT VIMAR 14594

### DISCUSSION THREAD

**Discussion by mkel84** [indomus/forum#2810] [ZigBee, Z-Wave, Thread, Matter]
Salve,
volevo chiedervi un aiuto sull'inserimento del modulo per la gestione delle tapparelle Vimar modello 14594 in Zigbee2MQTT. Sono riuscito ad associare il modulo a Zigbee2MQTT tramite la chiavetta sonoff zigbee 3.0 installata su un raspberry. L'associazione è corretta ma i valori sono invertiti, la tapparella è aperta e sull'applicativo è nello stato "CLOSE", se è chiusa è nello stato "OPEN".
Ho visto la funzione "Invert cover" nella sezione "Impostazioni (specifiche)" ma non è cambiato nulla.
Avete suggerimenti?

**Comment by snakuzzo** [comment]
_invert_cover: Inverts the cover position, false: open=100,close=0, true: open=0,close=100 (default false). The value must be true or false_

Se hai fatto tutto bene nel `configuration.yaml` di Z2M dovresti trovare questo (al netto dell'indirizzo del device)...

```yaml
devices:
  '0x00158d0002bfc307':
    invert_cover: true
```

Se è così, il test fallo prima dalla webgui di zigbee2mqtt, perché magari su HA non si è allineato per qualche motivo.

**Comment by mkel84** [comment]
Provato ma non va, devo provare a fare un’installazione pulita direttamente sul sistema operativo, da docker mi sembra poco performante

**Reply by snakuzzo** [reply]
Ma cosa dici ? :D
Non c'entra niente docker o non docker...inoltre, se sai cos'è, con docker è tutto performante allo stesso modo di un'installazione direttamente su OS.
Se hai questo problema, di sicuro non dipende da docker :)

**Reply by mkel84** [reply]
Lo so che il problema non è docker ma è di configurazione, ho solo notato che molto spesso zigbee2mqtt mostra degli errori di comunicazione con i dispositivi e volevo tentare di capire se installando tutto sul SO la situazione migliora. Altrimenti mi tocca reinstallare il firmware dell'antenna.

**Reply by snakuzzo** [reply]
Non cambia nulla
